### PR TITLE
Add support for BETAFPV_900_TX_MICRO and BETAFPV_2400_TX_MICRO

### PR DESCRIPTION
**Pull request by brandonrc**
Not sure why BetaFPV labeled their targets micro but that is what I am using for this PR. 

https://github.com/BETAFPV/ExpressLRS/blob/48b2b206c91461a43e6eca0983ce5ba6da8182c2/src/include/targets.h#L550

### CODE REVIEW

File: src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h [hunk 1]
```diff
@@ -12,12 +12,17 @@
 #define GPIO_PIN_RST            14
 #define GPIO_PIN_RX_ENABLE      27
 #define GPIO_PIN_TX_ENABLE      26
-#define GPIO_PIN_OLED_SDA       -1
-#define GPIO_PIN_OLED_SCK       -1
+#define GPIO_PIN_OLED_CS 15
+#define GPIO_PIN_OLED_RST 16
+#define GPIO_PIN_OLED_DC 17
+#define GPIO_PIN_OLED_MOSI 32
+#define GPIO_PIN_OLED_SCK 33
```
Comment (JyeSmith) [inline src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h:19]: Tabbing

File: src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h [hunk 2]
```diff
@@ -12,12 +12,17 @@
...
 #define GPIO_PIN_RCSIGNAL_RX    13
 #define GPIO_PIN_RCSIGNAL_TX    13
 #define GPIO_PIN_LED_WS2812     15
 #define GPIO_PIN_FAN_EN         17
+#define HAS_OLED
+#define HAS_OLED_SPI
```
Comment (JyeSmith) [inline src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h:25]: Should HAS_OLED and HAS_OLED_SPI be in the target .ini?

Comment (brandonrc) [reply src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h]: So, I think these should be in the targets.. they are hardware specific and I think we also have some packback stuff in a few of those other target.h files. 

I moved them up to a nice `// Any device features` section.

**Comment by CapnBry** [comment]
* The TX_DEVICE_NAMEs are now just DEVICE_NAME
* I'm not too keen on including one target definition then including another target definition that #undefs a bunch of things to redefine them. It should probably be handled in the way like things based on `DIY_2400_TX_ESP32_SX1280_E28` are where the child includes the parent, or just fully specify the defines in the child instead of trying to override the parent?

**Comment by brandonrc** [comment]
> * The TX_DEVICE_NAMEs are now just DEVICE_NAME
> * I'm not too keen on including one target definition then including another target definition that #undefs a bunch of things to redefine them. It should probably be handled in the way like things based on `DIY_2400_TX_ESP32_SX1280_E28` are where the child includes the parent, or just fully specify the defines in the child instead of trying to override the parent?

Yea, I would rather have one file for all the targets. Like a betafpv.h file. So that we do not have to worry about 50 dang header files for each target. when you could have an easy `define` statement. 

The issue is now I would need to have what? Three header files? 
```
betafpv_2400_TX_base.h
betafpv_2400_TX_regular.h?
betafpv_2400_TX_micro.h
```

**Comment by brandonrc** [comment]
The TX_DEVICE_NAMEs are now just DEVICE_NAME

@CapnBry when did this change? Because in the code everything says TX_DEVICE_NAMES on my branch.

**Review by CapnBry — APPROVED** [review]

**Review by pkendall64 — APPROVED** [review]

**Pull request merged**